packages/components/src/context-tag/context-tag.tsx

```diff
@@ -179,37 +228,48 @@ const ContextTag = (props: Props) => {
               src={props.network.src}
               name={props.network.name}
             />
-            <Label size={size}>{props.network.name}</Label>
+            <Label size={size} textSize={textSize}>
+              {props.network.name}
+            </Label>
           </>
         )
       }
       case 'collectible': {
         return (
           <>
+            {/* fixme: not an avatar but a resized image; see https://www.figma.com/file/IBmFKgGL1B4GzqD8LQTw6n?node-id=1336:34320&mode=design#541620575 */}
```

Comment:
fixme: not an avatar but a resized image; see https://www.figma.com/file/IBmFKgGL1B4GzqD8LQTw6n?node-id=1336:34320&mode=design#541620575

Reply:
https://github.com/status-im/status-web/issues/472